internal/adapters/events/events_forwarder.go

```diff
@@ -46,20 +49,20 @@ func NewEventsForwarder(forwarderClient ports.ForwarderClient) *eventsForwarder
 }
 
 // ForwardRoomEvent forwards room events. It receives the room event attributes and forwarder configuration.
-func (f *eventsForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder entities.Forwarder) error {
+func (f *eventsForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder entities.Forwarder) (int32, error) {
```

Comment:
Why do we return an `int32` as the code? Since we are only adding for metrics and we send the metric as a string, makes sense for me to return it as a string.